src/modules/BootGRUB2.rb

```diff
@@ -152,6 +152,9 @@ def Propose
         partition = prep_partitions.first
         if partition
           BootCommon.globals["boot_custom"] = partition
+          BootCommon.globals["boot_boot"] = "false"
+          BootCommon.globals["boot_root"] = "false"
+          BootCommon.globals["boot_mbr"] = "false"
```

Comment:
ok, then go ahead

btw, my test completed and everything worked but the prep partition wasn't tagged as active; so it's still not correct...